### ISSUE TITLE
 No module named 'glide_text2im'

### ISSUE THREAD

**Issue by dfrankle2** [afiaka87/pyglide#1]
does this only work on windows?

**Comment by CounterFlow64** [comment]
> 

No, I got it installed easily on linux. Just open the terminal and follow the suggested commands without the (.venv) stuff after the first line, I will post it here for convenience. 

python3 -m venv .venv
source .venv/bin/activate
python -m pip install -r requirements.txt
git clone https://github.com/openai/glide-text2im.git
cd glide-text2im/
python -m pip install -e .
 cd ../

But I can't run anything in my experience, I just get "AttributeError: 'SpacedDiffusion' object has no attribute 'plms_sample_loop"
